supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/MachineConstraints.scala

```diff
@@ -20,6 +20,7 @@ object MachineConstraints {
       val customMachineType =
```

Comment:
It reads a little strange that the comment on line 19 only mentions Cascade Lake when something similar is now also happening with AMD Rome

Reply:
Updated comment with regards to AMD Rome

Reply:
test cases added for CPU specified 81 => 96 and 128 => 96.